cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/vsphere/vsphere-mc-common/VsphereCredentialsStep.tsx

```diff
@@ -35,6 +37,9 @@ export function VsphereCredentialsStep(props: Partial<StepProps>) {
     const [connectionErrorMessage, setConnectionErrorMessage] = useState('');
     const [datacenters, setDatacenters] = useState([] as VSphereDatacenter[]);
```

Comment:
This looks a little strange to me. do you mean useState<VSphereDatacenter[]>([])?

Reply:
Agreed!